integration_tests/src/janus.rs

```diff
@@ -174,29 +58,27 @@ impl<'a> Drop for Janus<'a> {
         // (log export is a no-op for non-containers: when running tests against a cluster, we
```

Comment:
nit: this comment should be updated, probably to drop the parenthetical sentence -- the non-container variant is now gone.